### PR TITLE
Add bash test target to run generate code before the test_util

### PR DESCRIPTION
**Pull request by farhankamalkhan-10p**
Remove doc.go because generate code will run before the test using the bash script
Update Makefile to run the bash script with the provided NIST profiles in test_util/artifacts
Update Makefile to exclude the go files in test_util since they will be run using the bash script
Add NIST profiles and catalog in test_util/artifacts
Add oscal-profile-schema and oscal-prose-module for validator_test.go

Signed-off-by: farhankamalkhan <farhankamalkhan@gmail.com>

### CODE REVIEW

**Review by mohuk — APPROVED** [review]

**Comment by mohuk** [comment]
LGTM

File: test_util/src/methods.go [hunk 1]
```diff
@@ -157,7 +158,6 @@ func ProfileProcessing(parsedProfile *profile.Profile) map[string][]string {
 		if check == "Catalog" {
 
 			ProfileControls := ImportParsing(parsedProfile, parsedProfile.Imports[l].Href.Path)
-
 			catalog := dirName + "/" + save[len(save)-1]
```
Comment (asadullah-yousuf-10p) [inline test_util/src/methods.go]:
```suggestion
			catalogPath := dirName + "/" + save[len(save)-1]
```

File: test_util/src/methods.go [hunk 2]
```diff
@@ -141,11 +137,16 @@ func ProfileProcessing(parsedProfile *profile.Profile) map[string][]string {
...
 	for l := 0; l < len(parsedProfile.Imports); l++ {
-		println("Import:", parsedProfile.Imports[l].Href.Scheme+"://"+parsedProfile.Imports[l].Href.Host+parsedProfile.Imports[l].Href.Path)
-		//println("Import:", parsedProfile.Imports[l].Href.Path)
-		dirName, err := DownloadCatalog(parsedProfile.Imports[l].Href.Scheme + "://" + parsedProfile.Imports[l].Href.Host + parsedProfile.Imports[l].Href.Path)
-		if err != nil {
-			log.Fatal(err)
+		println("Import:", parsedProfile.Imports[l].Href.String())
+		var dirName string
+		if !strings.Contains(parsedProfile.Imports[l].Href.String(), "http") {
+			dirName = "test_util/artifacts/"
```
Comment (asadullah-yousuf-10p) [inline test_util/src/methods.go]:
```suggestion
			dirName = baseArtifactsDir // constant
```

File: test_util/src/methods.go [hunk 3]
```diff
@@ -157,7 +158,6 @@ func ProfileProcessing(parsedProfile *profile.Profile) map[string][]string {
 		if check == "Catalog" {
 
 			ProfileControls := ImportParsing(parsedProfile, parsedProfile.Imports[l].Href.Path)
```
Comment (asadullah-yousuf-10p) [inline test_util/src/methods.go]:
```suggestion
			ProfileControls := ParseImport(parsedProfile, parsedProfile.Imports[l].Href.Path)
```

**Review by asadullah-yousuf-10p — CHANGES_REQUESTED** [review]
Just a few changes.

**Comment by farhankamalkhan-10p** [comment]
Closing to create PR on docker/oscalkit

**Pull request closed without merging**